app.R

```diff
@@ -1,21 +1,21 @@
 #BM: it's nice for other users if we auto-install missing packages
 #for them
```

Comment:
we should discuss -- would like to make sure i understand how the necessary libraries will be loaded.  i'm guessing this is only relevant for running the app locally.